xmtp_mls/migrations/2024-05-15-145138_new_schema/up.sql

```diff
@@ -0,0 +1,104 @@
+-- Your SQL goes here
+CREATE TABLE openmls_key_store(
+    key_bytes BLOB PRIMARY KEY NOT NULL,
+    value_bytes BLOB NOT NULL
+);
+
+CREATE TABLE identity_inbox(
+    "inbox_id" text NOT NULL,
```

Comment:
Should we add sequence id to identity? 

```suggestion
    "inbox_id" text NOT NULL,
    "sequence_id" BIGINT NOT NULL,
```

Reply:
Probably, but lets do that as a separate PR